PocketKit/Sources/PocketKit/Item/Item+Extensions.swift

```diff
@@ -73,4 +73,14 @@ public extension Item {
     var hasArticleComponents: Bool {
         article?.components.isEmpty == false
     }
+
+    var isCollection: Bool {
+        // TODO: Refactor when working on opening a collection tickets
+        guard let url = URL(string: givenURL), url.host == "getpocket.com",
+              url.pathComponents.count >= 3,
+              url.pathComponents[safe: 1] == "collections" else {
+            return false
+        }
+        return true
```

Comment:
This seems reflective of Android, which is nice. Is this just a Swift "port" of their logic?

Reply:
I had to increase the count by 1, but otherwise it's a port over

Reply:
![image](https://github.com/Pocket/pocket-ios/assets/6743397/8fbe187f-8bf1-4bf9-a458-a327d0712382)